lib/improver/tests/utilities/test_rescale.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+"""Unit tests for the rescale function from rescale.py."""
+
+import unittest
+import numpy as np
+import StringIO
+import sys
+
+from iris.tests import IrisTest
+
+from improver.utilities.rescale import rescale
+from improver.tests.nbhood.nbhood.test_BaseNeighbourhoodProcessing import (
+    set_up_cube)
+from improver.tests.ensemble_calibration.ensemble_calibration.helper_functions\
+    import add_forecast_reference_time_and_forecast_period
+
+
+class Test_rescale(IrisTest):
+
+    """Test the utilities.rescale rescale function."""
+
+    def setUp(self):
+        """
+        Create a cube with a single non-zero point.
+        Trap standard output
+        """
+        self.cube = add_forecast_reference_time_and_forecast_period(
+            set_up_cube())
+        self.stdout = StringIO.StringIO()
+        sys.stdout = self.stdout  # Redirect standard out.
+
+    @staticmethod
+    def tearDown():
+        """Reset standard out stream."""
+        sys.stdout = sys.__stdout__  # Reset standard out.
+
+    def test_basic(self):
+        """Test that the method returns the expected array type"""
+        result = rescale(self.cube.data)
+        self.assertIsInstance(result, np.ndarray)
+
+    def test_debug(self):
+        """
```

Comment:
Following the style in https://github.com/metoppv/improver/wiki/Code-Style-Guide#unit-test-structure, I think it's preferable for docstrings to be 
```python
"""Words"""
```
rather than 
```python
"""
Words
"""
```

Reply:
Docstrings updated.